wallet/README.md

```diff
@@ -1,3 +1,6 @@
+Wallet - Technical details for development and debugging
```

Comment:
I'd say just "Technical details". If it were just for development, we'd rather rename the README a bit.